examples/ethernet_network.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"os"
+)
+
+func main() {
+	var (
+		ClientOV *ov.OVClient
+		eth_name = "eth1"
+		new_name = "eth77"
+		upd_name = "eth88"
```

Comment:
You can give more understandable examples rather than the name of the resources rather than these name like ethernet and updated_ethernet for more understandability.